components/Frontpage/WelcomeScreen.tsx

```diff
@@ -73,10 +76,46 @@ const RootContainer = styled('div')`
   overflow: hidden;
 `;
 
+const LiveHeader = styled.span`
+  position: relative;
+  display: inline-block;
+  border-radius: 5px;
+  color: #f05454;
+  font-weight: 900;
+  stroke: white;
+  font-size: 35px;
+  text-shadow: 3px 2px 3px rgba(0, 0, 0, 0.4);
+
+  -webkit-text-stroke-width: 0.75px;
+  -webkit-text-stroke-color: white;
+`;
+
+// Update the currentEvent-list every sec
+const intervalLength = 1000 * 1;
+
 const WelcomeScreen = ({ currentMetaData }: Props): JSX.Element => {
+  const [time, setTime] = useState(dayjs());
+  const [active, setActive] = useState(false);
+
   const startDate = dayjs(currentMetaData.startDate);
   const endDate = dayjs(currentMetaData.endDate);
 
+  // TODO: Double-check this logic
+  useEffect(() => {
+    const beforeEventStart = timeIsAfter({
+      time: time,
+      start: toDayjs(currentMetaData.startDate, '09:30:00'),
+    });
+
+    if (beforeEventStart) {
+      const interval = setInterval(() => setTime(dayjs()), intervalLength);
+      return (): void => clearInterval(interval);
+    }
+    if (!beforeEventStart) {
+      setActive(true);
+    }
+  }, [currentMetaData, time]);
+
```

Comment:
Why is this in a `useEffect`?

Reply:
The useEffect is to set the time which in turn sets `active`. I've refactored it to two seperate useEffects so that the interval is only activated at componentMount

Reply:
Ah, right. The interval needs to be there :ok_hand: